cli/build.rs

```diff
@@ -70,13 +70,13 @@ fn main() {
   let bundle_path = o.join("COMPILER_SNAPSHOT.js");
   let snapshot_path = o.join("COMPILER_SNAPSHOT.bin");
   let mut custom_libs: HashMap<String, PathBuf> = HashMap::new();
+  custom_libs
+    .insert("lib.window.d.ts".to_string(), c.join("js/lib.window.d.ts"));
+  custom_libs
+    .insert("lib.worker.d.ts".to_string(), c.join("js/lib.worker.d.ts"));
   custom_libs.insert(
-    "lib.deno_main.d.ts".to_string(),
-    c.join("js/lib.deno_main.d.ts"),
-  );
-  custom_libs.insert(
-    "lib.deno_worker.d.ts".to_string(),
-    c.join("js/lib.deno_worker.d.ts"),
+    "lib.window_or_worker_global_scope.d.ts".to_string(),
+    c.join("js/lib.window_or_worker_global_scope.d.ts"),
   );
   custom_libs.insert("lib.deno.d.ts".to_string(), c.join("js/lib.deno.d.ts"));
```

Comment:
possibly `lib.deno.ns.d.ts` so it is clear it is the Deno namespace? (I know I am questioning my own code)